### PR TITLE
adjust wget parser for improved performance

### PR DESCRIPTION
**Pull request by loredous**
Previous wget parser regex was causing extreme backtracking issues, leading to 5+ minute runtimes for large log files. Improved performance and updated tests to account for different data extraction.

### CODE REVIEW

File: code/parsers/wget.py [hunk 1]
```diff
@@ -33,29 +34,42 @@ class WgetParser(ParserBase):
...
+        )
+        self.output_extractor_regex = re.compile(
+            r"-O (.*?) "
         )
-        self.ip_extractor = re.compile("(Connecting to \S+ \(\S+\)\|(\S+)\|:\d+)")
-        self.filename_extractor = re.compile("(Saving to: [‘'](\S+)[’'])")
 
     def get_document_dependencies(self, document: str) -> List[ExtractedDependency]:
         dependencies = []
+        print(f'Started blocks - {time.time()}')
```
Comment (hippee-lee) [inline code/parsers/wget.py:48]: Did you want to keep the print statements in here for logging (also ending print stmt on line #50)?

**Review by hippee-lee — DISMISSED** [review]
lgtm - I left a question about the print statements but it doesn't need to block.

File: code/parsers/wget.py (hunk 1 above)
Comment (loredous) [reply code/parsers/wget.py:48]: Ah, good catch!

**Review by hippee-lee — APPROVED** [review]
lgtm - I left a question about the print statements but it doesn't need to block.

**Pull request merged**